### DISCUSSION TITLE
HackaLearn Korea 이벤트에 여러분을 환영합니다!

### DISCUSSION THREAD

**Discussion by justinyoo** [devrel-kr/HackaLearn#10] [0 공지사항]
## 👋 여러분 환영합니다!

이곳이 여러분들께시 챌린지를 완성하기 위해 필요한 모든 질문과 답변이 이루어지는 곳입니다. 챌린지를 진행하다가 막히는 부분이 있으면, 궁금한 부분이 있으면 언제든 이곳에 질문을 남겨주세요. Microsoft Learn Student Ambassadors (MLSA), GitHub Campus Experts (GCE), Microsoft Cloud Advocate (CA), 그리고 현장의 전문가 분들께서 여러분의 궁금한 점을 해결해 주실 겁니다.

0. [공지사항](/devrel-kr/HackaLearn/discussions/categories/0-공지사항)
1. [Q&A HackaLearn 운영](/devrel-kr/HackaLearn/discussions/categories/1-q-a-hackalearn-운영)
2. [Q&A 클라우드 스킬 챌린지](/devrel-kr/HackaLearn/discussions/categories/2-q-a-클라우드-스킬-챌린지)
3. [Q&A 일반](/devrel-kr/HackaLearn/discussions/categories/3-q-a-일반)
4. [Q&A 기획](/devrel-kr/HackaLearn/discussions/categories/4-q-a-기획)
5. [Q&A UX](/devrel-kr/HackaLearn/discussions/categories/5-q-a-UX)
6. [Q&A 프론트엔드](/devrel-kr/HackaLearn/discussions/categories/6-q-a-프론트엔드)
7. [Q&A 백엔드](/devrel-kr/HackaLearn/discussions/categories/7-q-a-백엔드)

여러분의 도전을 응원합니다!

**Comment by loreenkim** [comment]
챌린지를 참여하는 모든 분들 환영합니다! :) 
기획과 관련된 질문은 언제든지 대환영입니다~

**Comment by Lee-WonJun** [comment]
여기다가 질문하면 되는걸까요~

[SNS/블로그 포스팅 챌린지](https://github.com/devrel-kr/HackaLearn/blob/main/challenges/SOCIAL.md) 에서 Linkedin 은 불가능할까요?
